BUILDING.md

```diff
@@ -0,0 +1,39 @@
+# Building Protocol Buffers
```

Comment:
You should be explicit here that these are only instructions for building with bazel, as there are many other ways currently to build protobufs.

Reply:
We also have this version:
https://github.com/protocolbuffers/protobuf/blob/main/src/README.md

Reply:
There isn't an "official way to build protobufs". People choose how to use protobufs based on their project's needs. Most of the time the best way to use protobufs is to depend on a project that imports protobufs or download an already built executable. If someone directly wants to build protobufs they do it based on the tools they are currently/would like to use, which is why we need to support so many platforms (cmake, autotools, maven, bazel)